src/store/getters.js

```diff
@@ -32,18 +33,25 @@ export default {
       asset,
       accountId,
       useCache = true,
-      walletType = 'default',
-      index = 0
+      accountType = 'default',
+      accountIndex = 0
     }) => {
       const account = accountId ? getters.accountItem(accountId) : null
-      const accountType = account?.type || walletType
-      const accountIndex = account?.index || index
+      const _accountType = account?.type || accountType
+      const { chain } = cryptoassets[asset]
+      let derivationPath
+      if (account && account.derivationPath) {
+        derivationPath = account.derivationPath
+      } else {
+        derivationPath = getDerivationPath(chain, network, accountIndex, _accountType)
```

Comment:
Hmm, why would an account ever not have a derivation path?

Reply:
hehe agree, but we can use the ledger provider we don't have it
the account comes from the accountId  const account = accountId ? getters.accountItem(accountId) : null, but can be null so when is null we just get the derivation path the case for this is when we want to get the accounts from ledger

Reply:
added a change to ask just when is ledger and we don't have an existing account